README.md

```diff
@@ -69,34 +76,44 @@ For a complete list of available commands, run this command:
 chifra --help --verbose
 ```
 
-Next, let's see if you can get a block from your Ethereum node. Enter this command:
+Next, check that you can get a block from your Ethereum node. Enter this command:
 
 ```[shell]
 getBlock 100
 ```
 
-This should return valid JSON data for block 100 (type `getBlock --help` for more options on this tool.) If you get an error, such as this:
+This should return valid JSON data for block 100 (type `getBlock --help` for
+more options with this tool). If you get an error like this:
 
 ```[shell]
 Warning: The Ethereum RPC: 'http://localhost:8545' was not found. Quitting...
 ```
+
 You need to edit the file `~/.quickBlocks/quickBlocks.toml` to provide the URL of an available Ethereum RPC provider. Add this text:
 
 ```[toml]
 [settings]
 rpcProvider="<url-to-your-rpc-provider>
 ```
 
-Once you get `getBlock` to return JSON data, you are ready to test your installtion. While optional, one of the tools (`ethslurp`) requires an Etherscan key. Get an EtherScan API key now, then run:
+Once you get `getBlock` to return JSON data, you are ready to test your installation. In the `./build` directory, run this command:
```

Comment:
Instead of `getBlock` (the underlying tool), we should use `chifra blocks` in these examples.